src/odemis/acq/stitching/_registrar.py

```diff
@@ -499,9 +502,9 @@ def getPositions(self):
         tile_positions = []
         dep_tile_positions = []
 
-        registered_positions = self._assemble_mosaic()
+        self.registered_positions = self._assemble_mosaic()
```

Comment:
`registered_positions` doesn't really contain the positions, but the shifts (that's my mistake). Could you rename all occurrences (also in the other registrars) to something like `self.shifts`?